packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1632,3 +2128,294 @@ Specific property types support a subset of [JSON Schema validations](https://to
+
+| Property                       | Description                                | DataType                                          | Nullable | Unique | Readonly |
+|:-------------------------------|:-------------------------------------------|:--------------------------------------------------|:---------|:-------|:---------|
+| title                          | user-defined display name for the property | String                                            | FALSE    | FALSE  | FALSE    |
+| description                    | description of the property                | String                                            | TRUE     | FALSE  | FALSE    |
+| type                           | type of property                           | `string`, `boolean`, `number`, `integer`, `array` | FALSE    | FALSE  | FALSE    |
+| enum                           | enumerated value of the property           | array                                             | TRUE     | TRUE   | FALSE    |
+| oneOf                          | non-empty array of valid JSON schemas      | array                                             | TRUE     | TRUE   | FALSE    |
+
+##### Description Details
+
+* `enum`: The value of the property is limited to one of the values specified in the `enum` definition.  The list of values for `enum` has to be made up of unique elements.
+
+* `oneOf`: Okta only supports `oneOf` for specifying display names for an `enum`. Each schema has the following format:
+
+ ```json
+{
+  "const": "enumValue",
+  "title": "display name"
+}
+ ```
+
+In case `enum` is used in conjunction with `oneOf`, the set of enumerated values and their order must be kept.
+
+```json
+{"enum": ["S","M","L","XL"],
+  "oneOf": [
+    {"const": "S", "title": "Small"},
+    {"const": "M", "title": "Medium"},
+    {"const": "L", "title": "Large"},
+    {"const": "XL", "title": "Extra Large"}
+  ]
+}
+ ```
+
+`oneOf` is only supported in conjunction with `enum`, providing a mechanism to return a display name for the `enum` value.
+
+Okta has also extended [JSON Schema Draft 4](https://tools.ietf.org/html/draft-zyp-json-schema-04) with the following keywords:
+
+| Property      | Description                                     | DataType                                                                  | Nullable | Unique | Readonly |
+| :------------- | :----------------------------------------------- | :------------------------------------------------------------------------- | :--------- | :------ | :-------- |
+| required      | determines whether the property is required     | Boolean                                                                   | FALSE    | FALSE  | FALSE    |
+| scope         | determines whether an appuser attribute can be set at the individual or group level | `SELF`, `NONE`                        | FALSE    | FALSE  | TRUE     |
```

Comment:
This also seems to be a copy-paste mistake. ie . `appuser attribute`

Reply:
Yes. Fixed.